task/fbc-related-image-check/0.2/MIGRATION.md

```diff
@@ -0,0 +1,48 @@
+## Deprecation notice
+
+This task is deprecated, please remove it from your pipeline.
+Deprecation date: 2024-12-31
```

Comment:
This seems like a bit of an aggressive deadline considering there's not a lot of working days until that date?

Reply:
Same applies to the other tasks.

Reply:
(I'd avoid using a date that falls on a common holiday.)

Reply:
I can put it into January. I had this as a placeholder initially. What would be a better date, January 15th?

Reply:
Based on recent user feedback on the [verify-signed-rpm](https://github.com/konflux-ci/build-definitions/tree/main/task/verify-signed-rpms) task being required in EC, I'd suggest Jan 31.